test/newton1d.jl

```diff
@@ -0,0 +1,53 @@
+
+using IntervalArithmetic, IntervalRootFinding
+using ForwardDiff
+using Base.Test
+
+const D = IntervalRootFinding.derivative
+
+setprecision(Interval, Float64)
+float_pi = @interval(pi)
+
+setprecision(Interval, 10000)
+big_pi = @interval(pi)
+# Using precision "only" 256 leads to overestimation of the true roots for `cos`
+# i.e the Newton method gives more accurate results!
+
+half_pi = big_pi / 2
+three_halves_pi = 3*big_pi/2
+
+@testset "Testing newton1d" begin
+
+    f(x) = e^(x^2) - cos(x)
+    f′(x) = 2*x*e^(x^2) + sin(x)
```

Comment:
Write these as exp(x^2) and then they will work. In general you should use exp instead of e